File: aufgabe01/skleinealber/fizzbuzz.php
<?php
  // Test
  //print_r($argv);
  for($i=1;$i<=$argv[1];$i++){
    if($i%3==0 && $i%5!=0){echo("Fizz");}
    elseif($i%3!=0 && $i%5==0){echo("Buzz");}
    elseif($i%3==0 && $i%5==0){echo("FizzBuzz");}
    else{echo($i);}
    echo("\n");
  }
?>

File: aufgabe01/shoeffner/FizzBuzz.py
# -*- coding: utf8 -*-
import sys

def FizzBuzz(limit):
	for number in range(1, limit+1):
		retNum = ''
		if number % 3 == 0:
			retNum += 'Fizz'
		if number % 5 == 0:
			retNum += 'Buzz'
		yield str(number) if retNum == '' else retNum

if __name__ == '__main__':
	for number in FizzBuzz(int(sys.argv[1])):
		print(number)